server/src/controllers/eventInstanceController.ts

```diff
@@ -255,45 +313,101 @@ eventInstanceController.get('/:id', async (req: Request, res: Response) => {
  *           application/json:
  *             schema:
  *               $ref: '#/components/schemas/Error'
+ *       422:
```

Comment:
Any reason why 422 and not just using 400 since usually 400s handle the bad requests?